application/controllers/RegisterController.php

```diff
@@ -333,6 +346,12 @@ public function getTokenId($iSurveyId)
                 $this->aRegisterErrors[] = gT("This email address is already registered but email to that adress could not be delivered.");
             } else {
                 $this->sMailMessage = gT("The address you have entered is already registered. An email has been sent to this address with a link that gives you access to the survey.");
+                $now = dateShift(date("Y-m-d H:i:s"), "Y-m-d H:i", Yii::app()->getConfig('timeadjust'));
+                $delay = Yii::app()->getConfig('registrationEmailDelay');
+                if (!empty($oToken->sent) && $oToken->sent != "N" && dateShift($oToken->sent, "Y-m-d H:i", $delay) > $now) {
+                    /** @todo: Figure out a way to show a different message to the user */
```

Comment:
My opinion : update $this->sMailMessage = at this point. Because he was set some line before.

The fact it was not used in view are part of another tickekt

Reply:
Check now please.